README.md

```diff
@@ -9,31 +9,41 @@ JWST Calibration Pipeline
 
 Note
 ----
-Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
+Beginning with version 0.9.0, **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest stable version of the library, we recommend using [conda](https://conda.io/docs/index.html)
+and the [astroconda](https://astroconda.readthedocs.io) channel to install the
+software and using an conda environemnt for the installation:
 
-    % cd jwst
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda create -n jwst python=3
+    source activate jwst
+    conda install jwst
 
-    % git checkout stable
+To install the development version of the repository, we recommend created a new
+nvironment, using the [astroconda](https://astroconda.readthedocs.io) channel
+to install the dependencies, and then installing from the github repository:
 
-    % python setup.py install
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda create -n jwst python=3
+    source activate jwst
+    conda install jwst
+    git clone https://github.com/STScI-JWST/jwst.git
+    cd jwst
+    python setup.py develop
 
-The main development is on the **master** branch. To install the development version:
+If you will not need to update the code, then you can replace the last step with
+the following command:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+    python setup.py install
```

Comment:
The pipeline's dependency list is gigantic and requires a few non-python packages in order to run. Here's a way to install those dependencies using `--only-deps` (as of conda 4.4.10):

```
$ conda create -n jwst_devel --only-deps --override-channels -c http://ssb.stsci.edu/astroconda-dev -c defaults jwst
```